calcom/cal.com
# adds event types to kbar

## What does this PR do?

Adds Event Types to KBar.

Fixes #6196 

**Environment**: Production

<img width="598" alt="image" src="https://user-images.githubusercontent.com/26844/209882434-d21095f8-7f22-4e20-b7de-9b96aa0aa804.png">

## Type of change


## How should this be tested?

* Create event types
* Open kbar
* Check for event type entries

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
